BitSliceIndexing/bsi.go

```diff
@@ -497,3 +612,78 @@ func (b *BSI) NewBSIRetainSet(foundSet *roaring.Bitmap) *BSI {
 	wg.Wait()
 	return newBSI
 }
+
+func (b *BSI) Clone() *BSI {
```

Comment:
If this is public, it should be documented.

Reply:
Done.